src/components/tables/cells/EntityStatus.tsx

```diff
@@ -1,66 +1,90 @@
 import { Box, Tooltip, Typography } from '@mui/material';
+import { Shard } from 'data-plane-gateway/types/shard_client';
 import { useRouteStore } from 'hooks/useRouteStore';
-import { CSSProperties } from 'react';
-import { shardDetailSelectors, ShardStatus } from 'stores/ShardDetail';
+import { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+    defaultStatusColor,
+    shardDetailSelectors,
+    ShardStatusColor,
+    TaskShardDetails,
+} from 'stores/ShardDetail';
 
 interface Props {
     name: string;
 }
 
 function EntityStatus({ name }: Props) {
+    const [taskShardDetails, setTaskShardDetails] = useState<
+        TaskShardDetails[]
+    >([]);
+    const [compositeStatusColor, setCompositeStatusColor] =
+        useState<ShardStatusColor>(defaultStatusColor);
+    const [taskDisabled, setTaskDisabled] = useState<boolean>(false);
+
     const shardDetailStore = useRouteStore();
```

Comment:
Same as above - needs to be renamed as a hook.

Reply:
PENDING.